x-pack/docs/en/ilm/apis/explain.asciidoc

```diff
@@ -0,0 +1,78 @@
+[role="xpack"]
+[testenv="platinum"]
+[[ml-delete-forecast]]
+=== Delete Forecast API
```

Comment:
Copy and paste error, these doc needs to be updated to reflect the ILM explain API